shared/wallets/common/wallet-popup.js

```diff
@@ -99,9 +117,13 @@ const styles = Styles.styleSheetCreate({
 })
 
 export default compose(
-  renameProp('onClose', 'onCancel'),
-  withProps({
-    style: Styles.isMobile ? null : {height: 525},
-  }),
+  withProps((props: WalletPopupProps) => ({
```

Comment:
Had some issues with flow in this function (see comments). I think it has to do with recompose typing but I didn't dig too deep.